engine/src/main/scala/cromwell/engine/package.scala

```diff
@@ -93,8 +93,8 @@ package object engine {
     val materializeWorkflowDescriptorActor = actorSystem.actorOf(MaterializeWorkflowDescriptorActor.props())
 
     materializeWorkflowDescriptorActor.askNoTimeout(MaterializeWorkflow(id, sources))  map {
-      case MaterializationSuccess(workflowDescriptor) => workflowDescriptor
-      case MaterializationFailure(error) => throw error
+      case MaterializationWorkflowDescriptorSuccess(workflowDescriptor) => workflowDescriptor
+      case MaterializeWorkflowDescriptorFailure(error) => throw error
```

Comment:
These two classes don't seem to be named consistently... maybe `MaterializeWorkflowDescriptorFailure` and `MaterializeWorkflowDescriptorSuccess`?